pkgs/development/compilers/cudatoolkit/redist/overrides.nix

```diff
@@ -49,7 +49,7 @@ in
         if [ -z "\''${CUDAHOSTCXX-}" ]; then
           export CUDAHOSTCXX=${cc}/bin;
         fi
-        export NVCC_PREPEND_FLAGS+=' --compiler-bindir=${cc}/bin'
+        export NVCC_PREPEND_FLAGS+=' --compiler-bindir=${cc}/bin -Xfatbin=-compress-all'
```

Comment:
could we add a comment (in nix) clarifying what this does and why it's important?

Reply:
Will do!